byron/ledger/formal-spec/blockchain-interface.tex

```diff
@@ -254,7 +255,8 @@ \subsection{Update-proposals interface}
       {\left(
         \begin{array}{l}
           s_n\\
-          \var{dms}
+          \var{dms}\\
+          \var{t}
```

Comment:
This is a system level parameter, let's leave it as such. We're not likely to be changing it.

Reply:
Are you saying it shouldn't be in any STS'es environment, for example the way `k` is not in any environment any more?

Reply:
Yes

Reply:
There is `upAdptThd`, which is available in protocol parameters. I'm not completely sure what's happening here, i.e., how it is related to `t`. Can you help here?

Reply:
Removing the parameter in the separate PR is ok, but the incorrect changes to this document should be addressed before this PR is merged. There's no sense in merging something known to be incorrect and then having a separate PR to revert it.

Reply:
Just fixed this.